packages/foreman/python-websockify/python-websockify.spec

```diff
@@ -0,0 +1,189 @@
+%global pkgname websockify
+%global summary WSGI based adapter for the Websockets protocol
+Name:           python-%{pkgname}
+Version:        0.10.0
+Release:        3%{?dist}
+Summary:        %{summary}
+
+License:        LGPLv3
+URL:            https://github.com/kanaka/websockify
```

Comment:
Technically, the correct home is at https://github.com/novnc/websockify

Reply:
I swear I fixed that, but then I regenerated it and copied it from EPEL. I'll submit a PR to Fedora to fix it.